agents/summarizer/main.py

```diff
@@ -27,6 +28,31 @@
 Func[fixie_query_embed] says: answer from the resource
 A: answer from the resource
 
+Q: summarize https://some_url
+Ask Func[load_doc]: https://some_url
+Func[load_doc] says: #doc1
+Ask Func[fixie_query_embed]: Summarize this document #doc1
+Func[fixie_query_embed] says: ## Summary of the Document
+The excerpt provided discusses several topics:
+
+#### The Voynich Manuscript
+
+The Voynich manuscript is mentioned as an illustrated codex from the 15th century written in an unknown script and language that has never been decoded. Details include:
+
+- It has been forgotten and rediscovered multiple times over the last 500 years
+- It was the subject of letters between Egyptologists for hundreds of years before getting lost in a Jesuit library
+- Millennials and Gen Z are more interested in mysterious historical topics like the Voynich manuscript through the Internet and podcasts
+A: Here is a summary of https://some_url:
+The excerpt provided discusses several topics:
+
+#### The Voynich Manuscript
+
+The Voynich manuscript is mentioned as an illustrated codex from the 15th century written in an unknown script and language that has never been decoded. Details include:
+
+- It has been forgotten and rediscovered multiple times over the last 500 years
+- It was the subject of letters between Egyptologists for hundreds of years before getting lost in a Jesuit library
+- Millennials and Gen Z are more interested in mysterious historical topics like the Voynich manuscript through the Internet and podcasts
```

Comment:
Is it odd to have the response repeat itself?

Reply:
It's necessary because we're teaching the LLM to return the response from the query embed func back to the user.